src/Screens/Main/Chat/Input.tsx

```diff
@@ -25,9 +27,13 @@ export default ({ buddyId }: Props) => {
     dispatch({ type: 'newMessage/send/start', payload });
   };
 
-  const color = getBuddyColor(buddyId);
+  const { isPending, disableSending, messageContent } = useSelector(
+    newMessageState.getMessage(buddyId),
+  );
+
+  const [showPending, setShowPending] = React.useState(isPending);
```

Comment:
is ok to tie this to isPending? shouldn't the default be false? what is the reasoning?

Reply:
I ssee this triggering 1 extra re-render with no reason.

Reply:
Because if user send message and navigates away from the view and comes back. Right?